Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/test-settings/test-launch-dialog.html

```diff
@@ -7,10 +7,10 @@ <h1 class="pull-left" md-dialog-title>Share Test</h1>
     <div class="form-group">
       <label>Global URL</label>
       <div class="url-container">
-        <input type="text" class="form-control" [(ngModel)]="testLink" name="Test Link" />
-        <em class="material-icons" ngxClipboard [cbContent]="testLink" (cbOnSuccess)="copiedContent = true">content_copy</em>
+        <input type="text" class="form-control" [(ngModel)]="testLink" name="TestLink" id="TestLink" #TestLink onblur="this.select();" />
```

Comment:
But the content of the test box that is the test link is not remaining selected on eliminating this event(onblur) and method(select).